notebooks/post-startup.sh

```diff
@@ -8,9 +8,14 @@ set -o nounset
 set -o pipefail
 set -o xtrace
 
-# Send stdout and stderr from this script to a tmp file for debugging.
+# The linux user that JupyterLab will be running as. It's important to do some parts of setup in the
+# user space.
+readonly JUPYTER_USER="jupyter"
+
 cd /tmp || exit
```

Comment:
I think you don't need to be in the `/tmp` directory anymore.

Reply:
I think it's nice for the terra tar & other stuff from install to get cleaned up eventually, added a comment.